oshc/main/static/main/css/app.css

```diff
@@ -152,27 +162,80 @@ footer .container {
   margin-top: 15px;
 }
 
-.rectangle ul {
+.backToTop {
```

Comment:
Not a good practice to use such name. Refer to this [link](https://docs.google.com/document/d/1wA9paRA9cS7ByStGbhRRUZLEzEzimrNQjIDPVqy1ScI/pub#h.8aauk3v0ixn6) for more.